ush/create_convinfo.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+# create global_convinfo file for a given date
+date=$1
+# 2m ob use file (DO_GSISOILDA)
+if [ -z ${2} ]; then
```

Comment:
Prefer testing the argument count first:
```suggestion
if [[ $# -ge 2 && -z "${2}" ]]; then
```